docs/governance.rst

```diff
@@ -0,0 +1,253 @@
+Governance
+==========
+
+
+Overview
+--------
+
+Tedana is a relatively small open source project that requires specialized
+knowledge in multiple domains. This leads to several challenges. No one
+person on the current tedana development team has a combination of 
+available time plus expertise in collaborative software development, MRI
+physics, and advanced data processing methods to assume a primary project
+leader role. Even if such a person was interested, it may not benefit the
+project to overly rely on the existence of one person. We developed the
+following system the following goals in mind:
+
+- Promote open discussions.
+- Minimize the administrative burden.
+- Strive for consensus.
+- Provide a path for when consensus cannot be achieved.
+- Grow the community.
+- Maximize the `bus factor`_ of the project.
+- Acknowledge the leadership and time multiple people contribute to our
+  community without demanding more time than any individual can offer.
+  Dividing leadership responsabilies into multiple smaller roles also
+  makes it easier to encourage new people to take on a leadership role
+  without fearing that too much work will be required of them.
+
+This governance structure is a work-in-progress. We welcome both people
+who want to take on a leadership role as well as ideas for our to improve
+this structure.
+
+
+
+Steering Committee and Leadership Roles
+---------------------------------------
+
+Organizational Principles
+`````````````````````````
+The steering committee steers. The goal of the steering committee is to help
+guide the direction of the project. Decisions in the steering committee will 
+focus on how to present project issues to the broader community in a clear way
+rather than making project decisions without community input. 
+
+Issues or pull requests that can be reviewed by any community member can be
+reviewed and approved by the steering committee or steering committee members.
```

Comment:
I'm not sure that I understand this point ? Are we just trying to say that steering committee members do not have special approval above other community members ?

Technically, @tsalo and I have admin powers and we have _occasionally_ used them, although we try to avoid doing so as much as possible. I'd imagine steering committee members will likely have similar powers / process, no ?

Reply:
As it's written now, the steering committee has the power to make a PR more difficult to approve and to decide the best way to engage the broader community for significant PRs. Maintainers have the ability to decide to approve something that doesn't meet the standard criteria. In practice that means someone with admin powers should consult the maintainers before using those powers.
I just copied from the governance doc that we had been editing. This issue may be because of a disconnect caused by me writing the steering section and Taylor focusing on Maintainers in the governance section. Perhaps some of the powers Taylor gave to maintainers should rest with the steering committee. I'll read through this all again and welcome thoughts from others as well.